doc/run_with_mysql.md

```diff
@@ -0,0 +1,43 @@
+# How to connect MySQL with SQLFlow
+
+This tutorial explains how to connect SQLFlow with [MySQL](https://en.wikipedia.org/wiki/MySQL).
+
+## Connect Existing MySQL server
```

Comment:
should also update https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/run/docker.md#mysql ?